templates/default.py

```diff
@@ -71,11 +71,14 @@ def determine_pip_install_arguments():
 
     args.append("pip{pip_version}")
 
-    if include_setuptools(pre):
-        args.append("setuptools{setuptools_version}")
+    # We no longer install setuptools and wheel by default on Python 3.12+
+    # for parity with the `ensurepip` and `venv` modules.
+    if this_python < (3, 12):
+        if include_setuptools(pre):
```

Comment:
I'd prefer that we move this check into `include_setuptools` and `include_wheel`.